.changelog/2755.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+cli: Added check for empty deployUrl in output for release switch case
+CHANGELOG_GUIDE: Updated issue number to pull number 
```

Comment:
```suggestion
```

We don't include changelog guide updates in our changelog ❤️ I'll get this comitted, and then merge the PR!